### PR TITLE
Update survey.md

### PR DESCRIPTION
**Pull request by Cryptocxf**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Here is an updated survey paper on federated learning in the direction of **Security** and **Personalization** that can help us systematically learn about backdoor attacks and personalization in federated learning.

Add a new **PMLR20** paper: How to Backdoor Federated Learning. This is the first paper detailing the process of backdoor attacks in federated learning.

Add a new **TNNLS22** paper: Towards Personalized Federated Learning. This paper highlights the key ideas and challenges of personalized federated learning, with a unique classification and future outlook.

### CODE REVIEW

**Comment by Cryptocxf** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Jamie-Cui** [comment]
Thanks for your contribution, you may follow the example PR for formatting etc. :): https://github.com/secretflow/secretflow/pull/541

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md [hunk 1]
```diff
@@ -9,17 +9,44 @@
 
 - [Vertical Federated Learning](https://arxiv.org/pdf/2211.12814.pdf)
 
+- [Practical secure aggregation for privacy-preserving machine learning](https://dl.acm.org/doi/pdf/10.1145/3133956.3133982)[**CCS17**]
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/survey.md:12]: This paper seems not suitable to place in survey.md？

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md [hunk 2]
```diff
@@ -9,17 +9,44 @@
...
+
+- [How To Backdoor Federated Learning](http://proceedings.mlr.press/v108/bagdasaryan20a/bagdasaryan20a.pdf)[**PMLR20**]
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/survey.md:14]: Maybe move to Robust or Security

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md [hunk 3]
```diff
@@ -9,17 +9,44 @@
...
+## Privacy
+- [BatchCrypt: Efficient Homomorphic Encryption for Cross-Silo Federated Learning](https://www.usenix.org/system/files/atc20-zhang-chengliang.pdf)[**Usenix Security20**]
+
+- [Federated Learning with Differential Privacy: Algorithms and Performance Analysis](https://sci-hub.st/10.1109/tifs.2020.2988575)[**TIFS20**]
+
+- [Privacy-Enhanced Federated Learning Against Poisoning Adversaries](http://www.cs.unb.ca/~rlu1/paper/LiuLXCHL21.pdf)[**TIFS21**]
+
+- [ShieldFL: Mitigating Model Poisoning Attacks in Privacy-Preserving Federated Learning](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9762272)[**TIFS22**]
+
+## Roubust
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/survey.md:32]: typo

File: docs/awesome-pets/papers/applications/ppml/fl/survey.md [hunk 4]
```diff
@@ -9,17 +9,44 @@
...
+- [FLTrust: Byzantine-robust Federated Learning via Trust Bootstrapping](https://arxiv.53yu.com/pdf/2012.13995.pdf)[**NDSS21**]
+
+- [Privacy-Preserving Byzantine-Robust Federated Learning via Blockchain Systems](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9849010&tag=1)[**TIFS22**]
```
Comment (zhangxingmeng) [inline docs/awesome-pets/papers/applications/ppml/fl/survey.md:35]: Diito: not suitable in survey.md

**Comment by Cryptocxf** [comment]
Okay, I'll reorganize it

**Review by Cryptocxf — COMMENTED** [review]
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Here is an updated survey paper on federated learning in the direction of **Security** and **Personalization** that can help us systematically learn about backdoor attacks and personalization in federated learning.

Add a new **PMLR20** paper: How to Backdoor Federated Learning. This is the first paper detailing the process of backdoor attacks in federated learning.

Add a new **TNNLS22** paper: Towards Personalized Federated Learning. This paper highlights the key ideas and challenges of personalized federated learning, with a unique classification and future outlook.

@Jamie-Cui

**Comment by Jamie-Cui** [comment]
@zhangxingmeng Do you mind take another look at this PR? Also, @Cryptocxf you may add your github id to the contributors here: https://github.com/secretflow/secretflow/blob/main/docs/awesome-pets/awesome-pets.md

**Review by zhangxingmeng — APPROVED** [review]
LGTM

**Pull request merged**